packages/graphql-language-service-server/src/MessageProcessor.js

```diff
@@ -434,10 +442,8 @@ async function provideDiagnosticsMessage(
   return results;
 }
 
-function sendMessageIPC(message: any): void {
-  if (process.send !== undefined) {
-    process.send(message);
-  }
+function sendMessageIPC(message: any, writer: (message: string) => void): void {
+  writer.write(message);
 }
```

Comment:
I'm intentionally leaving it just in case we need some more processing for messages/writers later.